### ISSUE TITLE
Staging Review finding: Unnecessary `aria-label` text in service list item

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#106799]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** AEDP
**Product:** Design Patterns
**Feature:** Critical Information and Service List Item Components

## Findings details
**VA.gov Experience Standard - issue:** [User experiences predictable interactions with components and patterns.](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

In the [service list item component](https://design.va.gov/components/service-list-item), the heading link has an `aria-label` to add "Go to" before the visible link text --- eg. visible link text of "Education" is read to screen reader users as "Go to Education."

This `aria-label` doesn't provide any meaningful information to screen reader users, as the fact that it's a link already indicates that you'll be going to somewhere. But it also introduces a potential barrier for sighted voice command software users, as the command "Click Education" may fail if their software finds "Go to Education" in the accessibility tree and doesn't identify the partial match.

<details><summary>Link, screenshot or steps to recreate</summary>

Currently coded as:
```
<a href="https://www.va.gov/education" class="service-title-row" aria-label="Go to Education">
   <div class="header" tabindex="0">
      <va-icon class="icon school hydrated"></va-icon>
      <div class="name-and-chevron">
          <h3 class="service-name">Education</h3>
          <va-icon class="chevron-icon hydrated"></va-icon>
      </div>
   </div>
</a>
```

</details>

## Recommended action

Remove the `aria-label`.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by bellepx0** [comment]
Fixed, deleted `aria-label` in the heading link. There is an open [PR](https://github.com/department-of-veterans-affairs/component-library/pull/1563) in component-library.

<img width="501" alt="Image" src="https://github.com/user-attachments/assets/344b8b8d-f7d3-4b6f-8015-6c85352fafa9" />

**Comment by msbtterswrth** [comment]
confirmed in the PR, lgtm.
